index.js

```diff
@@ -3232,6 +3232,57 @@
   S.range =
   def('range', {}, [$.Integer, $.Integer, $.Array($.Integer)], range);
 
+  //# groupBy :: (a -> a -> Boolean) -> Array a -> Array (Array a)
+  //.
+  //. Splits its array argument into an array of arrays of equal,
+  //. adjacent elements. The equality of the elements is tested with
+  //. a curried function provided by the user in the first argument.
+  //.
+  //. The behaviour of groupBy can be surprising for functions that
+  //. aren't reflexive, transitive and symmetric
+  //. (see [equivalence][] relation).
```

Comment:
Let's make this part of the first paragraph. We can then drop "of groupBy" (relying on context). Finally, please add an Oxford comma after "transitive".